Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamPictureInPictureController.swift

```diff
@@ -103,36 +109,47 @@ final class StreamPictureInPictureController: NSObject, AVPictureInPictureContro
     public func pictureInPictureControllerWillStartPictureInPicture(
         _ pictureInPictureController: AVPictureInPictureController
     ) {
-        log.debug("Will start with trackId:\(track?.trackId ?? "n/a")", subsystems: .pictureInPicture)
+        Task { @MainActor in
```

Comment:
Is the log restricted to the main actor only? Seems like maybe we can avoid this?

Reply:
This is on purpose in order to provide the log when the actual operation is happening